### PR TITLE
Feat (#485) : added feature to close the active tab with middle mouse click

### PR DESCRIPTION
**Pull request by Sahilm416**
-> **feat #485** 
-> Added the functionality to close the active tab with middle mouse click
-> Middle click functionality is added inside the whole main section

### CODE REVIEW

File: packages/bruno-app/src/components/RequestTabPanel/index.js [hunk 1]
```diff
@@ -147,10 +167,43 @@ const RequestTabPanel = () => {
...
+                console.log('err', err);
+              });
+          }}
+        />
+      )}
       <div className="pt-4 pb-3 px-4">
         <QueryUrl item={item} collection={collection} handleRun={handleRun} />
       </div>
-      <section className="main flex flex-grow pb-4 relative">
+      <section className="main flex flex-grow pb-4 relative" onMouseDown={(e) => handleMouseDown(e)}>
```
Comment (martinsefcik) [inline packages/bruno-app/src/components/RequestTabPanel/index.js:206]: I tried the following applications (Firefox, Edge, Chrome, Posman) and all of them are closing tabs on middle click up event and not down.

**Comment by Sahilm416** [comment]
@martinsefcik  so i should trigger it onMouseUp right ?

**Comment by Sahilm416** [comment]
@martinsefcik will you please elaborate, we just need to add this functionality to everywhere in Main section or just on " X " button to close with OnMouseUp ??

**Comment by jakoss** [comment]
@Sahilm416 I'm pretty sure what he mean is to add middle click on the tab itself, not just the X button but whole tab. It's pretty much the default behavior for any app that has tabs now

**Comment by martinsefcik** [comment]
@Sahilm416 Yes, the whole tab, not X button. It would make no sense. I would just use left click on x button and not middle click in case it would be only on X button.

**Pull request closed without merging**